low-level/files/opt/seagate/sspl/conf/support.yaml

```diff
@@ -0,0 +1,3 @@
+bundle: 
+    create:
+      url: "sh /usr/bin/sspl_support_bundle"
```

Comment:
no lets not keep support bundle trigger in /usr/bin until asked for, it was wrong approach in cortxv1. give path from sspl deployment path here.

Reply:
Done